CMSgov/onemac
# OY2-5835 CHIP SPA RAI form

Stories:
- https://qmacbis.atlassian.net/browse/OY2-5835
- https://qmacbis.atlassian.net/browse/OY2-5836

Endpoint: https://d3pzlsq3vbiiff.cloudfront.net/

To test:
1. Verify that there are new Nightwatch test cases designed to test the CHIP SPA RAI form’s functionality and that they pass during the build.
2. Log in as `stateuseractive@cms.hhs.local` and submit a CHIP SPA (or find a CHIP SPA on the dashboard) and note its SPA ID./
3. Click the new link in the dashboard sidebar to respond to a CHIP SPA RAI and verify that you are taken to a new form with the correct file upload prompts.
4. Input your CHIP SPA ID from step 2. While doing so, verify that the ID validation is working properly.
5. Upload documents for the required fields and submit the form. Verify that the submission goes through successfully.
6. If you _really_ insist on testing OY2-5836, I wish you luck. You know what you're doing better than I do. If you don't insist so strongly, inspect the email logic in `services/app-api/changeRequest/CHIPSPARAI.js` and make sure it looks like the email templates in Confluence.
